tests/unit/store/playback.spec.js

```diff
@@ -256,6 +256,24 @@ describe("store/playback", () => {
         });
       });
     });
+    test("Given all axios requests are mocked to return 300, When stop_live_view is dispatched, Then the system status updates to be in the ERROR state", async () => {
+      mocked_axios.onGet(all_mantarray_commands_regexp).reply(300);
```

Comment:
since we already have unit tests for `axios_helper` function itself confirming that it works with a variety of error codes, I don't think it is necessary to have multiple different error codes tested here.  Do you think also testing multiple error codes here helps cover situations that the tests in `axios_helper.spec` might miss?

Reply:
As per my understanding axios_helpers.js could be axios_common_service.js and its just a common api among all actions. 

So any common API doesn't need a specific .spec file as its just an abstraction of common code, its not have any specific functionality to verify. there is no requirement of one more additional file axios_helpers.spec.js or axios_common_service.js as any way in the spec files of flask/waveform/playback the api is covered via either for 200 or other form of responses is a must requirement to verify.

Reply:
We want to use this in a similar way to `TextValidation`.  There is functionality inside TextValidation that we want to unit test rigorously.  But each time we have another piece of code use TextValidation we don't need to re-test all of the things that were already unit tested by the TextValidation spec file itself

Reply:
Removed the testcases for asserting the `status_ping_interval_id` and `waveform_ping_interval_id` for response of 300, 400 and 500 error response from flask.spec.js and playback.spec.js and trying to evolve a testcase to validate if id is reset to null if error 3xx,4xx and 5xx error are status, from additional testcases from axios_helper.spec.js

Reply:
it's important to have 1 test case for an error to make sure that the general functionality is wired up correctly.  I just meant that you don't need to rigorously test all the different error codes, since we already have unit tests in `axios_helper.spec` confirming that if that function is being used then it works for all error codes